Neuro/Tasks/TasksHandler.cs

```diff
@@ -20,43 +20,43 @@ public class TasksHandler : MonoBehaviour
     [HideFromIl2Cpp]
     public void UpdatePathToTask(PlayerTask task = null)
     {
-        if (!task) task = PlayerControl.LocalPlayer.myTasks.At(0);
+        bool isImpostor = NeuroPlugin.Instance.Impostor.isImpostor;
+        if (!task)
+        {
+            // skip fake tasks that cannot be completed (for instance, the red text as impostor)
+            foreach (PlayerTask t in PlayerControl.LocalPlayer.myTasks)
+            {
+                if (t.TryCast<ImportantTextTask>()) continue;
```

Comment:
This will mean that the pathfinding will go to solve sabotage tasks. Is this what you want?

If not, consider changing it to this:

```suggestion
                if (!t.TryCast<NormalPlayerTask>()) continue;
```